unittest.bat

```diff
@@ -49,6 +49,7 @@ FOR /D %%G IN ("*") DO (
         exit /B %ERRORLEVEL%
     )
     cd ..
+    :CONTINUE
```

Comment:
Is this still needed?

Reply:
You are right, this is now superfluous. I have removed it.